.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please remove add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~~not required~~, ~~covered~~ . Link --> #XYZ
```

Comment:
I agree, which is why I have [originally proposed a template](https://github.com/Mbed-TLS/mbedtls/pull/9008#issuecomment-2084739951) that had a justifiation after the checkboxes and the linkage. 

Saying why something is not where it should be, can be ingored by tooling but is very helpfull for the gatekeeper verifyin the validity of the template.

Reply:
I agree that we should indicate somehow that we require justification here. I like `~~not required because~~` better than the original `~~not required~~ (comments)` because it is more suggestive of the purpose. Maybe adding a semicolon (`~~not required because:~~`) would make it more likely that whoever edits them keeps the keywords.